pom.xml

```diff
@@ -35,7 +35,7 @@
     <junit.version>5.6.0</junit.version>
     <jacoco-maven-plugin.version>0.8.5</jacoco-maven-plugin.version>
     <maven-checkstyle-plugin.version>3.1.1</maven-checkstyle-plugin.version>
-    <checkstyle.version>8.32</checkstyle.version>
```

Comment:
Why did you downgrade checkstyle?

Reply:
otherwise the code doesn't compile and gives a dependency error

Reply:
thats your bad definitions on maven, old news, don't commit this